### PR TITLE
Adding list of classes taken and list of current classes to student

### PR DESCRIPTION
**Pull request by aminebechrifa**
Instead of adding new methods to the repository , The update method was changed so that it can be used directly for both adding classes from the usecase layer

### CODE REVIEW

**Comment by nuzurie** [comment]
Were you able to test it out? I don't think this will work.

File: Student/repository/student_create.sql [hunk 1]
```diff
@@ -6,6 +6,8 @@ CREATE TABLE IF NOT EXISTS public.student
     email character varying(64) COLLATE pg_catalog."default",
     general_info character varying(1024) COLLATE pg_catalog."default",
     school character varying(64) COLLATE pg_catalog."default",
+    classes_taken character varying(1024) COLLATE pg_catalog."default",
+    classes_taken character varying(1024) COLLATE pg_catalog."default",
```
Comment (car12lin12) [inline Student/repository/student_create.sql]: The duplicate of classes_taken should be enrolled_in or current_classes

File: Student/repository/student_repository.go [hunk 2]
```diff
@@ -110,7 +110,7 @@ func (r *studentRepository) Update(ctx context.Context, st *domain.Student) erro
 	}
 	defer tx.Rollback(ctx)
 
-	_, err = tx.Exec(ctx, update, st.ID, st.FirstName, st.LastName, st.Email, st.GeneralInfo, st.CreatedAt, st.UpdatedAt)
+	_, err = tx.Exec(ctx, update, st.ID, st.FirstName, st.LastName, st.Email, st.GeneralInfo,st.CurrentClasses,st.ClassesTaken, st.CreatedAt, st.UpdatedAt)
```
Comment (car12lin12) [inline Student/repository/student_repository.go:113]: is it okay if the order here (current classes, classes taken) doesnt match line 28 (classes taken, current classes)?

**Review by car12lin12 — CHANGES_REQUESTED** [review]
Nice job Amine! I have a similar question to Zubair, was this tested?

**Pull request closed without merging**